<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConductTestDialog</class>
 <widget class="QDialog" name="ConductTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>287</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_test">
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="list_test"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PushButton" name="btn_check">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>36</height>
        </size>
       </property>
       <property name="text">
        <string>Check</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>lib/ui/widgets_basic.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
